Entities/Structures/Door/SwingDoor.as

```diff
@@ -187,53 +190,52 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
 	if (customData == Hitters::bomb)
 		damage *= 1.3f;
 
-	CSprite @sprite = this.getSprite();
+	return damage;
+}
 
-	if (sprite !is null)
+void onHealthChange(CBlob@ this, f32 oldHealth)
+{
+	f32 hp = this.getHealth();
+	bool repaired = hp > oldHealth ? true : false;
```

Comment:
`bool repaired = (hp > oldHealth);` is semantically identical

Reply:
derp, fixed